lib/plugins/crowdsec/captcha.lua

```diff
@@ -65,16 +61,9 @@ function M.New(siteKey, secretKey, TemplateFilePath)
     M.SecretKey = secretKey
 
 -- for loop over core.backends
-
-    if core.backends["captcha_verifier"] == nil then
-      return "no verifier backend provided, can't use captcha"
-    end
-    for k, v in pairs(core.backends["captcha_verifier"].servers) do
-     	M.CaptchaProvider = utils.split(v.name, "_")[1]
-	M.CaptchaServerName = v.name
-    end
-    if M.CaptchaProvider == nil then
-      return "no verifier backend provided, can't use captcha"
+    M.CaptchaProvider = captchaProvider
```

Comment:
Captcha will no longer work in existing installations, without changing the configuration file and maybe haproxy version. I guess we just need to communicate appropriately about those changes, maybe change the version number also to reflect the breaking nature of this version.